src/Sarif.Viewer.VisualStudio/ResultTextMarker.cs

```diff
@@ -75,6 +75,14 @@ internal IVsWindowFrame NavigateTo(bool usePreviewPane)
                 }
             }
 
+            Uri uri;
+            if (File.Exists(this.FullFilePath) && Uri.TryCreate(this.FullFilePath, UriKind.Absolute, out uri))
```

Comment:
>out uri) [](start = 101, length = 8)

FYI, you can use an inlined declaration here for more concise syntax. Delete var decl above. use 'out Uri uri' in this call. :)